src/net/sourceforge/kolmafia/request/BrogurtRequest.java

```diff
@@ -47,7 +47,15 @@ public class BrogurtRequest extends CoinMasterRequest {
           GenericRequest.QUANTITY_PATTERN,
           null,
           null,
-          true);
+          true) {
+        @Override
+        public final boolean canBuyItem(final int itemId) {
+          if (7455 <= itemId && itemId <= 7457) {
```

Comment:
Don't use magic numbers.
Add the items to ItemPool.java.
Consider a switch statement, considering we're only talking 3 items here.